app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': true %>
+    <%= javascript_include_tag :cocoon %>
```

Comment:
Please follow instructions for rails 5.x https://github.com/nathanvda/cocoon#rails-31rails-4rails-5